nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/LoomServer.java

```diff
@@ -168,7 +169,19 @@ public int port(String socketName) {
 
     @Override
     public boolean hasTls(String socketName) {
-        return false;
+        ServerListener listener = listeners.get(socketName);
+        return listener != null && listener.hasTls();
+    }
+
+    @Override
+    public void reloadTls(String socketName, Tls tls) {
+        ServerListener listener = listeners.get(socketName);
+        if (listener == null) {
```

Comment:
I think this should throw an IllegalArgumentException, as the socket is not valid